packages/web/src/components/tipping/support/SupportingList.tsx

```diff
@@ -62,11 +63,10 @@ export const SupportingList = () => {
 
   return profile && rankedSupportingList.length > 0 ? (
     <div className={styles.container}>
-      <div className={styles.titleContainer}>
-        <IconTip className={styles.tipIcon} />
-        <span className={styles.titleText}>{messages.supporting}</span>
-        <span className={styles.line} />
-      </div>
+      <ProfileNavTitle
```

Comment:
maybe different name for this since it may be used outside profile one day.
...also... it having the name ProfileNavTitle but being used in ProfilePictureListTile seems a bit weird. down for suggestions

Reply:
IMO it's fine to tie this with Profile for now – maybe something like `ProfilePageNavSectionTitle`?